javascript.js

```diff
@@ -0,0 +1,66 @@
+// Main-Sections mit Navbar einblenden
+
+let mainElQuestions = document.querySelector('#main__questions')
+let mainElBookmarks = document.querySelector('#main__bookmarks')
+let mainElAddQuestion = document.querySelector('#main__create')
+let mainElProfile = document.querySelector('#main__profile')
+
+let buttonElHome = document.querySelector('#navbar__home')
+let buttonElBookmarks = document.querySelector('#navbar__bookmarks')
+let buttonElAddQuestion = document.querySelector('#navbar__add-question')
+let buttonElProfile = document.querySelector('#navbar__profile')
+
+let buttonElStart = document.querySelector('.intro__start')
```

Comment:
Instead of `let` you should use `const` in this case because the variables remain the same.

Reply:
Ok! Will use `const` instead of `let`.